src/main/java/frc/robot/subsystems/Intake/Intake.java

```diff
@@ -0,0 +1,42 @@
+package frc.robot.subsystems.Intake;
+
+import static frc.robot.GlobalConstants.ROBOT_MODE;
+
+import org.littletonrobotics.junction.Logger;
+
+import frc.robot.pioneersLib.subsystem.Subsystem;
+
+public class Intake extends Subsystem<IntakeStates>{
+    IntakeIO io;
+    IntakeIOInputsAutoLogged inputs;
+    private static Intake instance;
+    
+    private Intake(IntakeIO io) {
+        super("Intake", IntakeStates.IDLE);
+        this.io = io;
+        inputs = new IntakeIOInputsAutoLogged();
+    }
+
+    @Override
+    protected void runState() {
+        Logger.recordOutput("Intake State", getState());
```

Comment:
this logs output to the "recorded outputs" dir in ascope, you should do SUBSYSTEM_NAME/ + "Intake State" and store subsystem name in constants. Same for all other logs